### PR TITLE
Make CONFIGURATION.md more newbie friendly

### PR DESCRIPTION
**Pull request by simnalamburt**
**[Click](https://github.com/simnalamburt/kime/blob/doc/docs/CONFIGURATION.md)** to see rendered CONFIGURATION.md

### CODE REVIEW

**Comment by Riey** [comment]
It seems all good but kime also read config from `XDG_CONFIG_DIRS` which is normally `/etc/kime` it should be mentioned

**Comment by adenosie** [comment]
the default config is at `/etc/kime/config.yaml`. copying from here would be better than creating a new config file

**Comment by simnalamburt** [comment]
Thanks for the feedback! I've updated the docs!

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -1,8 +1,43 @@
...
+  - D2Coding
+  - 15.0
+compose:
+  compose_choseong_ssang: true
+  decompose_choseong_ssang: false
+  compose_jungseong_ssang: false
+  decompose_jungseong_ssang: false
+  compose_jongseong_ssang: false
+  decompose_jongseong_ssang: false
+```
```
Comment (Riey) [inline docs/CONFIGURATION.md:25]: Now default_config.yaml is exists in docs folder so this is unnessary

**Pull request merged**